apps/src/templates/rubrics/rubrics.module.scss

```diff
@@ -370,6 +370,57 @@ h2.studentName {
   }
 }
 
+// Start of rubrics redesign
+.learningGoalsContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: space-between;
+  align-self: stretch;
+  border-radius: 4px;
+  border: 1px solid $neutral_dark40;
+  background: $neutral_white;
+}
+
+.learningGoalButton {
+  background: none;
+  border: none;
+}
+
+.learningGoalsHeader {
+  display: flex;
+  flex-direction: row;
+  height: 40px;
+  padding: 0 1rem 0 2rem;
+  justify-content: space-between;
+  align-items: center;
+  align-self: stretch;
+  line-height: 154%;
+  position: relative;
+}
+
+.learningGoalsHeaderLeftSide {
+  @include main-font-semi-bold;
+  display: flex;
+  align-items: left;
+  //gap: 16px;
```

Comment:
Should this be removed?

Reply:
Leaving it for now because there is more design work that needs to be done on this and other components. That's from the previous version of the component.